templates/navigationTitle.jsx

```diff
@@ -2,12 +2,17 @@ import React from 'react';
 
 export default function NavigationTitle(props) {
   const {
-    title
+    title,
+    _useCourseTitle
   } = props;
 
   return (
 
-    <div className='navigation-title__inner' dangerouslySetInnerHTML={{ __html: title }} />
+    <div
+      className='navigation-title__inner'
+      dangerouslySetInnerHTML={{ __html: title }}
+      aria-level={!_useCourseTitle ? 1 : null}
```

Comment:
I think we'll need to include `role="heading"` too.

Reply:
Yep, this is done.